types/newrelic/index.d.ts

```diff
@@ -579,3 +579,836 @@ export function withLlmCustomAttributes<T>(
 export function setLlmTokenCountCallback<T>(
     cb: (...args: any[]) => T,
 ): T;
+
+/**
+ * @see https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration
+ */
+export interface NewRelicConfig {
```

Comment:
Everything in this package is New Relic related; does it need the `NewRelic` prefix?